app/components/markdown/markdown_link.js

```diff
@@ -10,7 +10,8 @@ import CustomPropTypes from 'app/constants/custom_prop_types';
 export default class MarkdownLink extends PureComponent {
     static propTypes = {
         children: CustomPropTypes.Children.isRequired,
-        href: PropTypes.string.isRequired
+        href: PropTypes.string.isRequired,
+        onLongPress: PropTypes.func.isRequired
```

Comment:
can we add this as a defaultProp and remove the `Required`?

I know this is being used in Channel Info to display the channel header